### ISSUE TITLE
Can't install aws-amplify from npm

### ISSUE THREAD

**Issue by 0x54321** [aws-amplify/amplify-js#9930]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-js/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-js/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### JavaScript Framework

Next.js

### Amplify APIs

Authentication

### Amplify Categories

auth

### Environment information

<details>

```
 System:
    OS: macOS 12.3.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 528.64 MB / 32.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.16.1 - /usr/local/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 6.14.14 - /usr/local/bin/npm
  Browsers:
    Chrome: 101.0.4951.64
    Firefox: 99.0.1
    Safari: 15.4
  npmPackages:
    @babel/core: ^7.16.0 => 7.16.0 
    @babel/eslint-parser: ^7.16.3 => 7.16.3 
    @babel/preset-env: ^7.16.4 => 7.16.4 
    @babel/preset-react: ^7.16.0 => 7.16.0 
    @material-ui/core: ^4.12.3 => 4.12.3 
    @material-ui/lab: ^4.0.0-alpha.46 => 4.0.0-alpha.46 
    @sentry/browser: ^6.15.0 => 6.15.0 
    @sentry/tracing: ^6.15.0 => 6.15.0 
    babel-loader: ^8.2.3 => 8.2.3 
    clean-webpack-plugin: ^0.1.17 => 0.1.19 
    css-loader: ^6.5.1 => 6.5.1 
    eslint: ^8.3.0 => 8.3.0 
    eslint-plugin-prettier: ^4.0.0 => 4.0.0 
    eslint-plugin-react: ^7.27.1 => 7.27.1 
    eslint-plugin-react-hooks: ^4.3.0 => 4.3.0 
    eslint-watch: ^8.0.0 => 8.0.0 
    js-yaml: ^4.1.0 => 4.1.0 
    match-sorter: ^2.2.0 => 2.3.0 
    moment: ^2.29.1 => 2.29.1 
    path: ^0.12.7 => 0.12.7 
    prettier: ^2.5.0 => 2.5.0 
    react: ^17.0.2 => 17.0.2 
    react-autosuggest: ^9.4.3 => 9.4.3 
    react-bootstrap: ^0.31.5 => 0.31.5 
    react-bootstrap-date-picker: ^5.1.0 => 5.1.0 
    react-bootstrap-table: ^4.3.1 => 4.3.1 
    react-datepicker: ^0.60.2 => 0.60.2 
    react-dom: ^17.0.2 => 17.0.2 
    react-hot-loader: ^4.13.0 => 4.13.0 
    react-select: ^1.3.0 => 1.3.0 (2.4.4)
    react-table: ^6.11.5 => 6.11.5 
    uuid: ^3.4.0 => 3.4.0 
    voucher-code-generator: ^1.1.1 => 1.1.1 
    webpack: ^5.64.4 => 5.64.4 
    webpack-cli: ^4.9.1 => 4.9.1 
    webpack-php-manifest: 0.0.8 => 0.0.8 
  npmGlobalPackages:
    @aws-amplify/cli: 7.6.5
    jshint: 2.13.1
    n: 8.0.2
    npm: 7.21.0


```

</details>

### Describe the bug

When I try to install aws-amplify via npm, it gives the following error:
`No matching version found for @aws-amplify/analytics@5.2.8.`

### Expected behavior

The package should install!

### Reproduction steps

npm i aws-amplify

### Code Snippet

```javascript
// Put your code below this line.

```

### Log output

<details>

```
// Put your logs below this line


```

</details>

**Comment by seanmarc** [comment]
Also unable to install aws-amplify
npm ERR! notarget No matching version found for @aws-amplify/analytics@5.2.8.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

Please help!

**Comment by michaelbarone** [comment]
I am experiencing the same issue currently.  was working this morning, but will not install dependencies without error.

**Comment by Ashish-Nanda** [comment]
Hi all,

We are aware of the issue and are working to address it right now as part of a new release for the package. We'll post an update once its resolved.

In the meantime you can install version `4.3.22` explicitly which should work correctly.

```npm i aws-amplify@4.3.22```

**Comment by Ashish-Nanda** [comment]
We've just published a new version `4.3.24` to resolve this issue and have verified the fix on our end.

Please let us know if you are still facing issues.

**Comment by manics** [comment]
I've opened a follow-up issue: https://github.com/aws-amplify/amplify-js/issues/9934
